packages/lesswrong/components/sequences/BooksProgressBar.tsx

```diff
@@ -0,0 +1,67 @@
+import { Components, registerComponent } from '../../lib/vulcan-lib';
+import React from 'react';
+import { Link } from '../../lib/reactRouterWrapper';
+import { postGetPageUrl } from '../../lib/collections/posts/helpers';
+import classNames from 'classnames';
+
+const styles = (theme: ThemeType): JssStyles => ({
+  root: {
+    display: "inline"
+  },
+  post: {
```

Comment:
rename to something clearer about what is this

Reply:
done